### DISCUSSION TITLE
Add the possibility to use grafanas default grouping in a nested policy

### DISCUSSION THREAD

**Discussion by nekater2** [grafana/grafana#68322] [Alerting]
I want to disable grouping [...] for my root policy while still sending all alerts in one group (default/null) for my nested policy.
So I configured a root policy with Group By: [...] and a nested policy with null (which is the default).
When doing this grafana inherits the grouping from the root policy [...] instead of using null and therefore puts each alert into an own group instead of putting all alerts into one. 

This is the expected behavior. But I still want to be able to use the default grouping in a nested policy

**Comment by konrad147** [comment]
Hey,
Unfortunately, it's not currently possible. Prometheus Alertmanager (Grafana Alerting needs to be compatible with it) allows defining `[]` as a single group only for root policy.
For nested policies `[]` and `undefined` means inherit from the parent

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82598. Feel free to continue the discussion around this there. Thank you!
